internal/pkg/gitdirty/gitdirty.go

```diff
@@ -23,6 +23,24 @@ func init() {
 	githubStyleHttpRemoteRegexp = regexp.MustCompile(`http[s]?:\/\/(.*?\..*?)\/(.*)`) // Example: https://git.test/testorg/testrepo.git
 }
 
+// GitInstalled checks if the command-line tool `git` is installed
+func GitInstalled() bool {
```

Comment:
Yes! Nice addition.

Reply:
Can Zen this up:

```
_, err := exec.LookPath("git")
return err == nil
```

(my university called this "boolean zen": https://courses.cs.washington.edu/courses/cse143/16au/style/bad-boolean-zen.html)